testcontainers-common/src/main/java/com/playtika/test/common/properties/CommonContainerProperties.java

```diff
@@ -37,25 +43,59 @@
 
 @Validated
 @Data
+@ConfigurationProperties("embedded.containers")
```

Comment:
Extending properties isn't that easy, respectively overriding members...
I think it could work without it, have to run all tests and change the new one.

Reply:
I'm not sure I follow you, to be honest, but to stay generic, this class should not have configuration properties annotation, so all extensions should force their own namespace for properties.

Reply:
I'm rewriting the new test accordingly, then I have to test all projects if it works.